### PR TITLE
file structure

### PR DESCRIPTION
**Pull request by MohammadAlhalaq**
Relates #1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
```
Comment (fadeomar) [inline package.json:14]: add shortcut for scripts we will use it in the project please

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "auto-complete-mmmf",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```
Comment (mossa-Sammer) [inline package.json:6]: add scripts for tape, tap-spec, nodemon

**Review by fadeomar — APPROVED** [review]

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <title>Project</title>
+
+    <!-- font library && font-awesome-->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <!-- google-font -->
+    <link href="https://fonts.googleapis.com/css?family=Darker+Grotesque:300,400,500,600,700,800,900&display=swap"
+        rel="stylesheet">
+    <!-- css files -->
+    <link rel="stylesheet" href="../public/CSS/root.css">
```
Comment (mossa-Sammer) [inline public/index.html]: make the css file name in lowercase

File: src/router.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const router = (req, res) => {
+
+}
+
+module.exports = router;
```
Comment (mossa-Sammer) [inline src/router.js]: insert a blank line

File: src/server.js [hunk 5]
```diff
@@ -0,0 +1 @@
+const router = require('./router');
```
Comment (mossa-Sammer) [inline src/server.js:1]: insert a blank line

**Review by mossa-Sammer — APPROVED** [review]

**Pull request merged**